.github/CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# CODEOWNERS are automatically assigned as possible reviewers to new PRs.
+
+# Global owners (also need to be duplicated in later rules)
+* @bertramakers @LucWollants
```

Comment:
Is this also not done here https://github.com/cultuurnet/stoplight-docs-errors/pull/7/files#diff-3d36a1bf06148bc6ba1ce2ed3d19de32ea708d955fed212c0d27c536f0bd4da7R4?